experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -6,6 +6,40 @@
 {% block title %}{{ experiment.name }}{% endblock %}
 
 {% block main_content %}
+  <!-- Invalid Pages Warnings Card -->
+  {% if experiment.get_invalid_pages %}
+    <div class="alert alert-danger" role="alert">
+      <p class="mb-1">
+        Before this experiment can be reviewed or launched, all required fields must be completed. Fields on the
+        <strong>
+          {% for page in experiment.get_invalid_pages %}
+            {% if page == "overview" %}
+              <a href="{{ experiment.get_update_overview_url }}?show_errors=true"
+                 rel="noopener noreferrer">Overview</a>
+            {% elif page == "branches" %}
+              <a href="{{ experiment.get_update_overview_url }}?show_errors=true"
```

Comment:
Oh yeah branches doesn't exist yet, but instead of linking to overview, just leave the href empty or something so we don't accidentally forget and leave it pointing to overview 🙏